generated/nitclk/__init__.py

```diff
@@ -6,7 +6,18 @@
 
 from nitclk.errors import DriverWarning   # noqa: F401
 from nitclk.errors import Error     # noqa: F401
```

Comment:
minor: why are there different number of spaces before the `#`?

Reply:
At one point they all lined up. Since they no longer do, and some lines have variables in them so we don't know how many spaces are required, made them all `<space><space>#<space>`